eclass/kernel-build.eclass

```diff
@@ -99,6 +99,14 @@ IUSE="+strip"
 # key in PEM format to use. Must be specified if MODULES_SIGN_KEY
 # is set to a path of a file that only contains the private key.
 
+# @ECLASS_VARIABLE: KERNEL_GENERIC_UKI_CMDLINE
+# @USER_VARIABLE
+# @DESCRIPTION:
+# If KERNEL_IUSE_GENERIC_UKI is set, this variable allows setting the
+# built-in kernel command line for the UKI. If unset, the default is
+# root=/dev/gpt-auto-root ro
+: "${KERNEL_GENERIC_UKI_CMDLINE:=root=/dev/gpt-auto-root ro}"
```

Comment:
I think you can use quoting inside `{}` here, that would make it a little more readable.

Reply:
> I think you can use quoting inside `{}` here, that would make it a little more readable.

Yes this appears to work, fixed!